python/tests/tests.py

```diff
@@ -214,13 +214,18 @@ def test_template(self):
 
     def test_pdu_len(self):
         enc = self.bser_mod.dumps(1)
-        self.assertEqual(len(enc), self.bser_mod.pdu_len(enc))
+        # Note: bser.pdu_len returns the PDU length AND the BSER version,
```

Comment:
Since this is confusing enough to warrant a comment, maybe rename the function to something more descriptive? :)